docs/source/deprecations.rst

```diff
@@ -2,8 +2,8 @@
 Deprecation timeline
 ====================
 
-From 3.3.0 to 4.0.0
-==========================
+From 3.3.0 to 4.0.0 (unreleased)
+===================
```

Comment:
docs won't build. the title underline should be as long as the title itself :/

Reply:
fixed!